content/1-introduction-2-review.md

```diff
@@ -0,0 +1,636 @@
+## Relevant works
+
+
+### 20th social science {#sec:intro-review-social}
+
+Social science has concerned itself with the stability, temporal evolution, and spatial variations and regularities of cultures since the start of its discipline.
+Émile Durkheim already, in his seminal study [-@durkheim_suicide:_2012], was looking at the regularities of suicide rates over the years, and the correlation of those rates with a partition of society into religion-related groups.
+The continuity of suicide over time, and the links between suicide rate and social group, he argued, suggest we should study suicide as a *social fact*:
```

Comment:
Oui, "the links between suicide rate and social group" ⟺ "le fait que des variables sociales (liées in fine à l'anomie) influencent son taux", oder? Pas assez clair tu dirais ?